src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props

```diff
@@ -4,5 +4,6 @@
 		<_JavaInteropReferences>Java.Interop;System.Runtime</_JavaInteropReferences>
 		<DependsOnSystemRuntime Condition=" '$(DependsOnSystemRuntime)' == '' ">true</DependsOnSystemRuntime>
 		<CopyNuGetImplementations Condition=" '$(CopyNuGetImplementations)' == ''">true</CopyNuGetImplementations>
+		<_YieldDuringExecution Condition="'$(_YieldDuringExecution)' == ''">true</_YieldDuringExecution>
```

Comment:
If the intention of this property is to allow "normal" developers to set it, i.e. it can/should be specified in `TheirAppProject.csproj`, then this shouldn't have a `_` prefix, it should instead be `$(AndroidYieldMSBuildDuringExecution)`.